### PR TITLE
Add draft of "Symmetric API Testing" (Advent 2015, Dec 26th)

### PR DESCRIPTION
**Pull request by ChimeraCoder**
Here is my draft for the post scheduled for Dec. 26th.

### CODE REVIEW

File: content/advent-2015/symmetric-api-testing-in-go.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
++++
+author = ["Aditya Mukerjee"]
+date = "2015-12-26T00:00:00-05:00"
+title = "Symmetric API Testing"
+series = ["Advent 2015"]
+draft = true
```
Comment (mattfarina) [inline content/advent-2015/symmetric-api-testing-in-go.md]: You can remove the `draft = true`. It's draft until merged so the pull request process handles that.

**Comment by mattfarina** [comment]
Overall this post looks good. What needs to happen for you to consider it ready to go instead of draft?

File: content/advent-2015/symmetric-api-testing-in-go.md (hunk 1 above)
Comment (ChimeraCoder) [inline content/advent-2015/symmetric-api-testing-in-go.md]: Ah, thanks!

**Comment by ChimeraCoder** [comment]
Thanks Matt - I made the small changes myself, so we're good to go as far as I'm concerned!

**Comment by mattfarina** [comment]
Thanks for your post. It's now live at https://blog.gopheracademy.com/advent-2015/symmetric-api-testing-in-go/

**Pull request merged**